### ISSUE TITLE
SG, EG supply grille extract grille

### ISSUE THREAD

**Issue by jgunstone** [theodi/BDNS#190]
for asset tagging

**device/asset description**
general terms for supply grille and extract grille

**device/asset name**

asset_description | asset_abbreviation | dbo_entity_type | ifc_class | ifc_type
-- | -- | -- | -- | --
grille/diffuser/louvre -   supply grille | SG |   | IfcAirTerminal | NOTDEFINED
grille/diffuser/louvre -   extract grille | EG |   | IfcAirTerminal | NOTDEFINED

**device/asset documentation**
https://www.swegon.com/siteassets/_product-documents/room-units/_en/airbourne-room-units/sd/swegon-wall-and-ceiling-grille-lnnn00398-1.0-web.pdf

**Comment by pisuke** [comment]
I think this an air terminal box, so the description should be changed

**Comment by pisuke** [comment]
What about:

asset_description | asset_abbreviation | dbo_entity_type | ifc_class | ifc_type
-- | -- | -- | -- | --
air terminal - air supply grille | ASG |   | IfcAirTerminal | NOTDEFINED
air terminal - air extract grille | AEG |   | IfcAirTerminal | NOTDEFINED

**Comment by pisuke** [comment]
We should see if we can keep these more generic, like AT, air terminal

**Comment by jgunstone** [comment]
I reviewed this and the MF head of mechanical is happy with AT for air terminal. 
we discussed using the [optional label suffix](https://github.com/theodi/BDNS/blob/master/BDNS_Specification_naming_syntax.md#optional-label-suffix) to indicate supply/extract. 

I think that will be a desire to be able to specify more detail in the future: 
i.e. 
GR - Grill
LV - Louver
JET - jet nozzle 

but for now I'll just aim to merge AT
